common/src/main/java/org/esbtools/message/admin/common/config/EMAConfiguration.java

```diff
@@ -0,0 +1,176 @@
+package org.esbtools.message.admin.common.config;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public final class EMAConfiguration {
```

Comment:
IMHO (and this may be out of scope of this refactoring), but instead of static getters I would make the configuration an interface, and have an implementation that expects a JSONObject to provider that configuration. Then either have a default constructor look for config in the same place, or a CDI `@Provider` (< hence not wanting to call our service API Provider... Provider is already a thing :)).

There's a few things I like about that personally. First, by having an interface, you make it clear what kinds of configuration EMA expects, without all the implementation detail cluttering that. Second, you allow for flexibility in implementations. Tests for instance don't have to provide the JSON if they don't want to, or it opens the door for other configuration methods.

In general, state accessed via static methods I think is better moved to an injected singleton (managed by a IoC container like CDI or Spring or w/e).